jobs/ENTD-006-FIQL-serialize-ast.md

```
@@ -491,3 +491,27 @@ than reading the review comment did, which is the argument for making it the
 default response to any doc-accuracy finding rather than a reaction to being
 told twice.
 
+### [Implementer] A behavioural diff against the base commit found what apidiff could not
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)